src/main/java/net/snowflake/ingest/utils/Constants.java

```diff
@@ -36,6 +36,7 @@ public class Constants {
   public static final String SNOWFLAKE_OAUTH_TOKEN_ENDPOINT = "/oauth/token-request";
   public static final String PRIMARY_FILE_ID_KEY =
       "primaryFileId"; // Don't change, should match Parquet Scanner
+  public static final String ASSIGNED_FULL_FILE_NAME_KEY = "assignedFullFileName";
```

Comment:
@sfc-gh-atoth `assignedFullFileName` is what we're embedding into the sdk now, let us know if you have a newer name to embed!

Reply:
Looks good.